shims/spark32/src/main/scala/org/apache/gluten/sql/shims/spark32/Spark32Shims.scala

```diff
@@ -189,8 +189,15 @@ class Spark32Shims extends SparkShims {
 
   def getFileStatus(partition: PartitionDirectory): Seq[FileStatus] = partition.files
 
+  def isFileSplittable(
+      relation: HadoopFsRelation,
+      filePath: Path,
+      sparkSchema: StructType): Boolean = true
+
   def isRowIndexMetadataColumn(name: String): Boolean = false
 
+  def findRowIndexColumnIndexInSchema(sparkSchema: StructType): Int = -1
+
```

Comment:
What does `-1` mean?

Reply:
`>=0` means found the index of `row index` column in schema, `-1` means not found